src/helpers/utils.ts

```diff
@@ -6,7 +6,7 @@ const ERROR_CODES: Record<string, number> = {
   'Invalid Request': -32600,
   PROPOSAL_NOT_FOUND: 404,
   PROPOSAL_NOT_CLOSED: -40004,
-  PENDING_GENERATION: -40010,
+  PENDING_GENERATION: 201,
```

Comment:
Looks like 201 is when file is already created, not pending https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201 🤔 

I think we should use 202 instead https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
> the request has been accepted for processing

Reply:
Correct, fixed